pkg/cluster/managed.go

```diff
@@ -47,13 +52,27 @@ func (c *Cluster) testClusterDB(ctx context.Context) (<-chan struct{}, error) {
 // start starts the database, unless a cluster reset has been requested, in which case
 // it does that instead.
 func (c *Cluster) start(ctx context.Context) error {
+	resetFile := filepath.Join(c.config.DataDir, "db", "reset-flag")
 	if c.managedDB == nil {
 		return nil
 	}
 
 	if c.config.ClusterReset {
+		if _, err := os.Stat(resetFile); err != nil {
+			if !os.IsNotExist(err) {
+				return err
+			}
+		} else {
+			return fmt.Errorf("cluster-reset was successfully performed, "+
```

Comment:
Nit: This can be a single line string. Go style doesn't care at all about archaic 80 char line length stuff.

Reply:
fixing